website/src/get-started/quickstart-py.md

```diff
@@ -207,9 +207,41 @@ This runner is not yet available for the Python SDK.
 ```
 
 After the pipeline completes, you can view the output files at your specified
-output path. For example, if you specify `/dir1/counts` for the `--output`
-parameter, the pipeline writes the files to `/dir1/` and names the files
-sequentially in the format `counts-0000-of-0001`.
+output path. For example, if you specify `outputs/part` for the `--output`
+parameter, the pipeline writes the files to `outputs/` and names the files
+sequentially in the format `part-0000-of-0001`.
+
+Here is how a basic implementation of WordCount looks like.
```

Comment:
same comments as what I have in the Go walkthrough. I'm also concerned that in this case, the mix of going back and forth between running a real .py file and using a colab (and the source code does not match) is going to cause confusion. What about moving the Colab to the bottom of the page after Inspect results, and have its own section, with the "Try the interactive Python WordCount example" etc. there.